core/controllers/classifier_test.py

```diff
@@ -151,10 +152,12 @@ def setUp(self):
                 u'answers': [u'a2', u'a3']
             }
         ]
-        self.job_id = classifier_services.create_classifier_training_job(
```

Comment:
@anmolshkl why are we removing this function if it is used in the code? I thought it wasn't being used anywhere and was a dead code. We should keep it as it is if it is being used.

Reply:
Oops my mistake. I thought this was code but it is test case. This is correct. Sorry for misinterpretation.